### DISCUSSION TITLE
使用附魔金苹果作为服务器流通货币

### DISCUSSION THREAD

**Discussion by RMSCA** [YuanYuanOwO/Minecraft-Tianji-Server#27] [Ideas]
当前经济系统容易出现通货膨胀的问题，而附魔金苹果有产量低，不可再生的特性，满足了稀缺性，属于硬通货，减少货币供应量可以控制通货膨胀，至于与服务器其他的货币的关系，可以考虑汇率挂钩？（有待商讨）
当前服务器货币有玩家可以无限刷钱的缺陷，虽然玩家每天可以获得的金币数量有限，但是这种方法只是治标不治本，该出现的问题迟早还是会出现，而工匠点数则是完全由抽奖的形式获得，而对玩家而言，最重要的货币就是工匠点数，个人感觉这种方式极为不靠谱。
此外，也可以限制服务器货币发行量，在货币发行到一定数量时，服务器收购资源时支付给玩家的钱可以使用玩家买入资源时支付给服务器的钱，玩家自己运营的商店则可以增加税收的机制，确保服务器货币流动

有什么想到的点子后续会继续往上加

**Comment by YuanYuanOwO** [comment]
https://github.com/YuanYuanOwO/Minecraft-Tianji-Server/issues/1

QuickShop税收会在近期重新开启
